docs/user/viewer-preferences.md

```diff
@@ -77,3 +77,7 @@ for i in range(40):
 with open("output.pdf", "wb") as output_stream:
     writer.write(output_stream)
 ```
+
+(The names beginning with a slash character are part of the PDF file format. They are
+included here to aid to anyone searching PyPDF documentation
```

Comment:
```suggestion
included here to aid to anyone searching pypdf documentation
```

Please mind the capitalization: `pypdf` should always be written completely lowercase everywhere.